AsyncDisplayKit/ASDisplayNode.mm

```diff
@@ -448,7 +448,8 @@ - (void)dealloc
   _flags.isDeallocating = YES;
 
   // Synchronous nodes may not be able to call the hierarchy notifications, so only enforce for regular nodes.
-  ASDisplayNodeAssert(_flags.synchronous || !ASInterfaceStateIncludesVisible(_interfaceState), @"Node should always be marked invisible before deallocating. Node: %@", self);
+  // FIXME: move UICollectionViewCell out of visible screen will trigger dealloc
+//  ASDisplayNodeAssert(_flags.synchronous || !ASInterfaceStateIncludesVisible(_interfaceState), @"Node should always be marked invisible before deallocating. Node: %@", self);
```

Comment:
There is an issue with `Display` state upon moving a cell out of window. The system deallocates before the state was cleaned up. Please advice.